server/src/services/queryValidator.service.types.ts

```diff
@@ -1,112 +1,186 @@
-export interface IRncpTCO {
-  result: Result
-  messages: Messages
-}
-
-interface Result {
-  _id: string
-  cfds: string[]
-  code_rncp: string
-  intitule_diplome: string
-  date_fin_validite_enregistrement: string
-  active_inactive: string
-  etat_fiche_rncp: string
-  niveau_europe: string
-  code_type_certif: any
-  type_certif: any
-  ancienne_fiche: string[]
-  nouvelle_fiche: any
-  demande: number
-  certificateurs: Certificateur[]
-  nsf_code: string
-  nsf_libelle: string
-  romes: Rome[]
-  blocs_competences: BlocsCompetence[]
-  voix_acces: any
-  partenaires: Partenaire[]
-  type_enregistrement: string
-  si_jury_ca: string
-  eligible_apprentissage: boolean
-  created_at: string
-  last_update_at: string
-  __v: number
-  rncp_outdated: boolean
-  releated: Releated[]
-}
-
-interface Certificateur {
-  certificateur: string
-  siret_certificateur: string
+export interface CertificationAPIApprentissage {
+  identifiant: Identifiant
+  intitule: Intitule
+  base_legale: BaseLegale
+  blocs_competences: BlocsCompetences
+  convention_collectives: ConventionCollectives
+  domaines: Domaines
+  periode_validite: PeriodeValidite
+  type: Type
+  continuite: Continuite
 }
 
-interface Rome {
-  rome: string
+interface Identifiant {
+  cfd: string
+  rncp: string
+  rncp_anterieur_2019: boolean
+}
+
+interface Intitule {
+  cfd: Cfd
+  niveau: Niveau
+  rncp: string
+}
+
+interface Cfd {
+  long: string
+  court: string
+}
+
+interface Niveau {
+  cfd: Cfd2
+  rncp: Rncp
+}
+
+interface Cfd2 {
+  europeen: string
+  formation_diplome: string
+  interministeriel: string
   libelle: string
+  sigle: string
+}
+
+interface Rncp {
+  europeen: string
 }
 
-interface BlocsCompetence {
-  numero_bloc: string
+interface BaseLegale {
+  cfd: Cfd3
+}
+
+interface Cfd3 {
```

Comment:
```suggestion
interface BaseLegaleCfd {
```

(a faire pour tous les autres, c'est plus compréhensible IMO)